### DISCUSSION TITLE
No input from Presonus Quantum ; with expansion interfaces .... Nothing Show on inputs ......Help , and TIA

### DISCUSSION THREAD

**Discussion by jbuecker** [jamulussoftware/jamulus#2827] [Technical Support & Help] [answered]
Im successful with the Other Jamming Platforms ; Can't get Jamulus to recognize inputs from a HIGH Input count...Please check!

**Comment by jbuecker** [comment]
Im successful with the Other Jamming Platforms ; Can't get Jamulus to recognize inputs from  a HIGH Input count...Please check!

**Reply by ann0see** [reply]
Did you start jamulus after pugging it in? Does it support 48kHz sample rate?

Did you read/try https://jamulus.io/wiki/Client-Troubleshooting?

**Comment by pljones** [comment]
Moving to a discussion for the raiser to supply useful diagnostics.

**Comment by jbuecker** [comment]
Definitely running @ 48Khz .  I will review the Troubleshooting article again..... FWIW ; in my DAW/Config , I have JAMULUS Attempting to listen on inputs 25 and 26 ... I have Persons Quantum with additional Mic Pre's routed through ADAT connections.... This configuration works fine for other similar apps, i.e. SONOBUS, JAMKAZAM, JAMTABA, SoundJack....
I have disabled and RE-ENABLED the microphone access for Jamulus in my Security Preference Pane.

**Comment by jbuecker** [comment, marked as answer]
UPDATE! ; Problem Resolved (I believe by re-enabling MICROPHONE access in System Preferences/Security )!
WAHOO!

**Reply by ann0see** [reply]
Great. Good to know!
